spec/unit/shoulda/matchers/active_model/validate_inclusion_of_matcher_spec.rb

```diff
@@ -886,6 +886,31 @@ def define_simple_model(
     def validation_matcher_scenario_args
       super.deep_merge(model_creator: :active_record)
     end
+
+    context 'against a polymorphic association' do
+      it 'matches when the subject configures validate_inclusion_of' do
+        define_model(:issue, severity_id: :integer, severity_type: :string) do
+          belongs_to :severity, polymorphic: true
+          validates_inclusion_of :severity_type, in: %w(Low Medium High)
+        end
+        define_model(:high)
+        define_model(:medium)
+        define_model(:low)
+
+        expect { validate_inclusion_of(:severity_type).in_array(%w(Low Medium High)) }.to match_against(Issue.new)
+      end
+
+      it 'does not match when subject does not set valid_inclusion_of' do
+        define_model(:issue, severity_id: :integer, severity_type: :string) do
+          belongs_to :severity, polymorphic: true
+        end
+        define_model(:high)
+        define_model(:medium)
+        define_model(:low)
+
+        expect { validate_inclusion_of(:severity_type).in_array(%w(Low Medium High)) }.not_to match_against(Issue.new)
```

Comment:
I intend to test the error message on this test.

Reply:
The error message at the moment: 
```
"After setting :severity_type to ‹\"Shoulda::Matchers::ActiveModel::ValidateInclusionOfMatcher::ArbitraryShouldaMatchersTestString\"›, the matcher expected the Issue to be invalid, but it was valid instead."
```

It seems too big for me. I'm working on it.